app/Http/Controllers/Api/CategoryController.php

```diff
@@ -0,0 +1,108 @@
+<?php
+
+namespace App\Http\Controllers\Api;
+
+use App\Models\Category;
+use App\Http\Controllers\Controller;
+use App\Http\Requests\Category\CategoryStoreRequest;
+use Exception;
+use Illuminate\Database\QueryException;
+use Illuminate\Http\Request;
+
+class CategoryController extends Controller
+{
+    /**
+     * Display a listing of the resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function index()
+    {
+        $category = Category::paginate(10);
+
+        return response()->json($category);
+    }
+
+    /**
+     * Show the form for creating a new resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function create()
+    {
+        //
+    }
```

Comment:
remove if not needed, if will be used in the future, you can retain.

Reply:
This function is removed. I'll keep in mind to remove the unused function.